.prettierignore

```diff
@@ -0,0 +1 @@
+*.hbs
```

Comment:
Should this file be in gitignore and not be pushed to repo?

Reply:
Everyone needs to use the same prettier settings so the repo stays consistent.